parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -52,7 +52,31 @@ write_files:
   content: !!binary |
     {{WrapAsVariable "systemConf"}}
 
+- path: /etc/apt/apt.conf.d/10periodic
```

Comment:
these two additive files are, I believe leftover from the master revert stuffs

Reply:
oops -- fixed